app/(main)/league/[leagueId]/entry/[entryId]/week/Week.tsx

```diff
@@ -100,7 +100,7 @@ const Week = ({ entry, league, NFLTeams, week }: IWeekProps): JSX.Element => {
 
       if (userWeeklyPickResults?.[user.id]?.[entry]) {
         const userPick = userWeeklyPickResults[user.id][entry].teamName;
```

Comment:
if this line is already getting the team name why do we do setUserPick(userPick.teamName) on line 103?